source/connect/mongoclient.txt

```diff
@@ -0,0 +1,210 @@
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. Any ref for authentication? Are we planning on making a page for authentication?
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, see
+       :ref:`pymongo-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`kotlin-sync-connection-options` for a full description of
+       these options.
+
+.. Is there relevant connection string content for kotlin sync to ref to? Only see for java sync.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string?tck=docs_driver_kotlin>` in the
+MongoDB Server documentation.
+
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, create a client. You can create a 
+client that uses your connection string and other client options by passing a 
+``MongoClientSettings`` object to the ``MongoClient.create()`` method.
+
+To instantiate a ``MongoClientSettings`` object, use the builder method to 
+specify your connection string and any other client options, and then call 
+the ``build()`` method. Chain the ``applyConnectionString()``` method to the 
+builder to specify your connection URI.
+
+You can set the Stable API version client option to avoid breaking changes when 
+you upgrade to a new server version. To learn more about the Stable API feature, 
+see the :ref:`Stable API page <kotlin-sync-stable-api>`. 
+
+The following code shows how you can specify the connection string and the 
+Stable API client option when connecting to a MongoDB deployment on Atlas 
+and verify that the connection is successful:
+
+.. literalinclude:: /includes/connect/mongoclient.kt
+   :start-after: start-connect-to-atlas
+   :end-before: end-connect-to-atlas
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Connect to a MongoDB Server or Replica Set
```

Comment:
I think everything in this section can be removed – it overlaps with the [Choose a Connection Target](https://github.com/mongodb/docs-kotlin-sync/pull/33) page currently in PR. I think this page should strictly cover creating a client, while the Choose a Connection Target page contains details about the different types of MongoDB instances you can connect to.